src/components/ebay-combobox/README.md

```diff
@@ -45,6 +45,7 @@ Name | Type | Stateful | Required | Description
 `disabled` | Boolean | Yes | No | sets the disabled attribute of the input
 `expanded` | Boolean | Yes | No | sets whether the listbox is expanded
 `autocomplete` | String | Yes | No | default is `none`; available values are `none` and `list`
+`aria-roledescription` | String | No | No | The role description for accessibility. Default text is set and will be in english. Pass this to override for different locales
```

Comment:
Can we drop the `aria-` suffix please? We'll come back and revisit all of these attr names in v6, but for now let's make it consistent with what we did with the `autocomplete` attribute.